README.md

```diff
@@ -1,124 +1,26 @@
 # Compose EVM Simplified
-This repo contains all the resources for deploying an EVM sidechain Node on mainnet or testnet.
 
----
-# Deployment
+This repository contains all the resources for deploying an EVM sidechain Node on mainnet or testnet. It can create either rpc nodes or forger nodes.
```

Comment:
How about his better: "This repository contains all the resources for deploying a forger or rpc EVM sidechain node on mainnet or testnet."